pkg/restapi/vault/operation/operations_test.go

```diff
@@ -101,6 +101,8 @@ func TestCreateVault(t *testing.T) {
 }
 
 func TestSaveDoc(t *testing.T) {
+	t.Skip()
```

Comment:
Is this supposed to be committed?

Reply:
No. PR is not ready, I have not added tests yet. I want to confirm the logic first.